src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -195,7 +195,7 @@
                         if (!this.initialised) {
                             return {max: 1, min: 0}
                         }
-                        return {max: this.colorIndicator.max, min: this.colorIndicator.min}
+                        return {max: this.colorIndicator?.max, min: this.colorIndicator?.min}
```

Comment:
> This seems irrelevant to this ticket but a quick fix for max error showing on initiation of Choropleth.

Not entirely comfortable with this - would be better in a new ticket, especially as it would likely benefit from a regression test and some guidance on manual testing. Especially as it can now return `{max: null, min: null}`, which looks a bit dubious...